### PR TITLE
RDKBACCL-282:SD Monolithic image support

### PR DESCRIPTION
**Pull request by hkumar343**
Reason for change:Bring up of sdcard flashing support to bpir4. 
Test Procedure: Mentioned in jira .
Risks: High

### CODE REVIEW

**Comment by mhughesacn** [comment]
Hi:
Binary (bin, img) files are not usually permitted in RDKM-hosted repositories.  Please can the img files be removed.

File: meta-rdk-mtk-bpir4/recipes-bsp/atf/files/mt7988/bpi-r4_sdmmc_fip.bin
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-bsp/atf/files/mt7988/bpi-r4_sdmmc_fip.bin:1]: binary file cannot be hosted inside a repo

File: meta-rdk-mtk-bpir4/recipes-bsp/atf/files/mt7988/bpi-r4_sdmmc_bl2.img
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-bsp/atf/files/mt7988/bpi-r4_sdmmc_bl2.img:1]: image/binary files cannot be hosted in a recipe

Comment (cpokuru) [reply meta-rdk-mtk-bpir4/recipes-bsp/atf/files/mt7988/bpi-r4_sdmmc_bl2.img:1]: host the files in artifactory.rdkcentral.com

File: meta-rdk-mtk-bpir4/recipes-bsp/atf/files/mt7988/bpir4_sd_image_creater.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+bootstart=17408
+bootsize=100
+rootsize=6600
+bootend=$(( ${bootstart}+(${bootsize}*1024*2)-1 ))
+rootstart=$(( ${bootend}+1 ))
+rootend=$(( ${rootstart} + (${rootsize}*1024*2) ))
+sudo sgdisk -o ${LDEV}
+sudo sgdisk -a 1 -n 1:34:8191 -A 1:set:2 -t 1:8300 -c 1:"bl2"           ${LDEV}
+sudo sgdisk -a 1 -n 2:13312:17407 -A 2:set:63   -t 2:8300 -c 2:"fip"            ${LDEV}
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-bsp/atf/files/mt7988/bpir4_sd_image_creater.sh:24]: sudo doesnt work on comcast build machines

File: meta-rdk-mtk-bpir4/recipes-bsp/atf/bootloader_prebuild.bb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+COMPATIBLE_MACHINE  = "^filogic$"
+
+inherit deploy
+
+PROVIDES += "atf_bootloader_prebuild"
+
+SRC_URI = "file://bpi-r4_sdmmc_bl2.img\
+           file://bpi-r4_sdmmc_fip.bin\
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-bsp/atf/bootloader_prebuild.bb:12]: keep it inside artifactory.rdkcentral.com

**Comment by cpokuru** [comment]
Going forward keep all WIP changes in local github.

**Comment by hkumar343** [comment]
new PR will be created with proper format.

**Pull request closed without merging**